aws/aws.go

```diff
@@ -363,19 +372,44 @@ func deleteCFStacks(cfClients map[string]cfClient, stacks []*types.Stack) []erro
 	errChan := make(chan error)
 
 	for r, s := range regionCFStacks {
-		go func(cfClient cfClient, region string, stacks []*types.Stack) {
+		go func(cfClient cfClient, rdsClient rdsClient, region string, stacks []*types.Stack) {
 			defer wg.Done()
 			for _, stack := range stacks {
 				if ctx.DryRun {
 					log.Infof("[AWS] Dry-run set, CloudFormation stack is not deleted: %s, region: %s", stack.Name, region)
 				} else {
+					dbStackResource, err := cfClient.DescribeStackResource(&cloudformation.DescribeStackResourceInput{
+						StackName:         &stack.Name,
+						LogicalResourceId: aws.String("DBInstance"),
+					})
+					if err != nil {
+						if !strings.Contains(err.Error(), "Resource DBInstance does not exist for stack") {
+							log.Errorf("[AWS] Failed to fetch the CloudFormation stack resources for stack: %s, err: %s", stack.Name, err)
+							errChan <- err
```

Comment:
Do we want to show the error in case it's not a DB CF stack Shouldn't we just ignore it?

Reply:
Nvm. I see we log it if it's not the expected error that it's not a DB stack.